### ISSUE TITLE
How to get the sparsed model?

### ISSUE THREAD

**Issue by zhaoyang-star** [neuralmagic/nm-vllm#125]
Very interesting project! I just saw the example below. I am curious that how to get the sparsed model based on one dense model? There are lots of sparse methods, such as SparseGPT, DejaVu, SliceGPT, etc,. Which method has nn-vllm supported?

```
from vllm import LLM, SamplingParams

model = LLM(
    "nm-testing/OpenHermes-2.5-Mistral-7B-pruned50",
    sparsity="sparse_w16a16",
    max_model_len=1024
)

sampling_params = SamplingParams(max_tokens=100, temperature=0)
outputs = model.generate("Hello my name is", sampling_params=sampling_params)
print(outputs[0].outputs[0].text)
```

**Comment by robertgshaw2-neuralmagic** [comment]
Hey @zhaoyang-star . These specific kernels are unstructured sparse, so `SparseGPT` is the algorithm we use to create the sparse models.

Here's a public example. There's a snippet in the model card that shows how to apply the algorithm. Ill swap the `nm-testing` model for this case.
- https://huggingface.co/neuralmagic/OpenHermes-2.5-Mistral-7B-pruned50

We also have a techniques called Sparse Finetuning, which uses SparseGPT + fine-tuning to recover accuracy
- https://arxiv.org/pdf/2310.06927.pdf

**Comment by jeanniefinks** [comment]
Hi @zhaoyang-star 
As some time has passed with no further comments, I am going to close out this thread. Feel free to re-open this if you want to continue the discussion. 

If you have not starred our NM repos and you feel inclined, we appreciate the support! Thanks ⭐ 
Regards, 
Jeannie / Neural Magic
